Makefile

```diff
@@ -1,11 +1,13 @@
-
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+APP_NAME ?= octopus
+IMG ?= $(APP_NAME):latest
+IMG-CI = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME):$(DOCKER_TAG)
 
 all: test manager
 
 # Run tests
 test: generate fmt vet manifests
+	dep status
```

Comment:
dep status on my machine takes 8-9 seconds. This is a lot. What about removing dep status from here and update all target to:
```
all: resolve test manager
```

Reply:
I replaced the `all` target with `validate` which is doing the whole validation, the `test` basically executing go tests.

The `validate` is gathered from [this](https://github.com/michal-hudy/community/blob/313a0c8cfda1c53830a585b7a611e118c145f602/sig-and-wg/sig-core/proposals/common-validator.md) proposal which was already accepted.